dockers/docker-bgp/Dockerfile

```diff
@@ -3,11 +3,15 @@ FROM docker-base
 COPY deps/quagga_*.deb /deps/
 RUN dpkg_apt() { [ -f $1 ] && { dpkg -i $1 || apt-get -y install -f; } || return 1; } &&        \
     dpkg_apt /deps/quagga_*.deb &&                                                              \
+    apt-get update &&                                                                           \
+    apt-get -y install -f python-jinja2 python-netaddr python-ipaddr python-lxml &&             \
     apt-get clean -y && apt-get autoclean -y && apt-get autoremove -y &&                        \
     rm -rf /deps
 
 COPY daemons /etc/quagga/
```

Comment:
it's already there. somebody add it in a previous version. 
and, it's actually not a template